packages/embark-ui/src/components/ContractOverview.js

```diff
@@ -196,13 +209,22 @@ class ContractFunction extends Component {
          <Collapse isOpen={this.state.functionCollapse} className="relative">
           <CardBody>
             {ContractFunction.isPayable(this.props.method) &&
-              <Form inline>
-                <Label for={this.props.method.name + '-value'} className="mr-2 font-weight-bold contract-function-input">ETH</Label>
+              <Form inline className="mb-1">
+                <Label for={this.props.method.name + '-value'} className="mr-2 font-weight-bold contract-function-input">Transaction value</Label>
                 <Input name={this.props.method.name}
                        id={this.props.method.name + '-value'}
                        onChange={(e) => this.handleChange(e, this.props.method.name + '-value')}
                        onKeyPress={(e) => this.handleKeyPress(e)}
+                       invalid={this.state.unitConvertError && this.state.unitConvertDirty ? true : null}
+                       valid={!this.state.unitConvertError && this.state.unitConvertDirty ? true : null}
                        />
+                {<FormFeedback valid={this.state.unitConvertError && this.state.unitConvertDirty ? null : true} tooltip>{this.state.unitConvertError ? this.state.unitConvertError : this.state.value} wei</FormFeedback>}
+                <Button close id="PopoverFocus" className="ml-2" type="button">
+                  <FontAwesome name="question-circle"/>
+                </Button>
+                <UncontrolledTooltip trigger="focus" placement="bottom" target="PopoverFocus">
+                  Enter a number followed by an ether unit, ie <code>0.5 ether</code> or <code>16 szabo</code>.<br/><br/>If a number is entered without a unit (ie <code>21000</code>), then <code>wei</code> is assumed.<br/><br/>Valid units include <code>noether</code>, <code>wei</code>, <code>kwei</code>, <code>Kwei</code>, <code>babbage</code>, <code>femtoether</code>, <code>mwei</code>, <code>Mwei</code>, <code>lovelace</code>, <code>picoether</code>, <code>gwei</code>, <code>Gwei</code>, <code>shannon</code>, <code>nanoether</code>, <code>nano</code>, <code>szabo</code>, <code>microether</code>, <code>micro</code>, <code>finney</code>, <code>milliether</code>, <code>milli</code>, <code>ether</code>, <code>kether</code>, <code>grand</code>, <code>mether</code>, <code>gether</code>, <code>tether</code>
```

Comment:
Wonder if it make sense to take those "unit labels", make them a property of this component and then generate this list based on the data. That way we avoid bloating this line too much. 🤔

Reply:
I moved these to a constant. Not sure if it made the line more pretty or not 🤔